builtin/logical/aws/path_config_root.go

```diff
@@ -230,25 +231,25 @@ func (b *backend) pathConfigRootWrite(ctx context.Context, req *logical.Request,
 		}
 
 		b.Logger().Debug("Registering rotation job", "mount", req.MountPoint+req.Path)
-		rotationID, err := b.System().RegisterRotationJob(ctx, rotationJob)
+		_, err = b.System().RegisterRotationJob(ctx, rotationJob)
```

Comment:
Not blocking this PR, but I would love to see RegisterRotationJob call ConfigureRotationJob for the user so that the plugin author doesn't need to make multiple calls to setup rotation.

Reply:
Good call! Updated the logic to move everything to a single call in `RegisterRotationJob`. This also helped simplify the external plugin GRPC interface for inputs so helped solve two issues. Thanks for the feedback!